crates/biome_cli/tests/snapshots/main_commands_rage/rage_ok.snap

```diff
@@ -26,6 +25,7 @@ Environment:
 
 Biome Configuration:
   Status:                       Not set
+  Path:                         unset
```

Comment:
How about to use `Not set` here to be consistent as the `Status` property also use the term above?